docs/reST/ref/context.rst

```diff
@@ -0,0 +1,52 @@
+.. include:: common.txt
+
+:mod:`pygame.context`
+======================
+
+.. module:: pygame.context
+    :synopsis: pygame module to provide additional context about the system
+
+| :sl:`pygame module to provide additional context on the system`
+
+**EXPERIMENTAL!** This API may change or disappear in later pygame releases. 
+If you use this, your code may break with the next pygame release.
+This is a new module, so we are marking it experimental for now.
+We probably won't have to change API, but we're keeping the possibility
+open just in case something obvious comes up.
+
+.. versionadded:: 2.1.3
+
+.. function:: get_pref_path
+
+   | :sl:`get a writeable folder for your app`
+   | :sg:`get_pref_path(org, app) -> path`
+
+   When distributing apps, it's helpful to have a way to get a writeable path,
+   because it's what apps are expected to do, and because sometimes the local
+   space around the app isn't writeable to the app.
+
+   This function returns a platform specific path for your app to store
+   savegames, settings, and the like. This path is unique per user and
+   per app name.
+
+   It takes two strings, ``org`` and ``app``. It then will figure out the
+   preferred path, **creating the folders referenced by the path if necessary**,
+   and returns a string containing the absolute path.
+
+    .. note::
+        Since the organization and app names can potentially be used as
+        a folder name, it is highly encouraged to avoid punctuation.
+        Instead stick to letters, numbers, and spaces.
```

Comment:
Looking at the SDL code it looks like if the app directory (or org directory) fails to create because it is an invalid string for a directory path it will raise an SDL error that says `Couldn't create a prefpath.` which I guess is _reasonably_ descriptive. SDL doesn't pass on errors like `ERROR_INVALID_NAME` so there isn't much we could do to distinguish that from any other potential error, I suspect that invalid directory name will probably be the most common error causing this function to fail. Perhaps it will mostly get used by slightly more advanced users who know about file systems, or we will need to strengthen the wording in the documentation later.